tools/config/__init__.py

```diff
@@ -600,8 +602,12 @@ def has_ram_regions(self):
 
     def deliver_into(self):
         if self.target.deliver_to_target:
-            label_dir = "TARGET_{}".format(self.target.deliver_to_target)
-            target_delivery_dir = join(DELIVERY_DIR, label_dir)
+            if self.target.delivery_dir:
+                target_delivery_dir = self.target.delivery_dir
```

Comment:
Can you show us an example of how it will work?

Reply:
see the updated release notes